### PR TITLE
Documentation content additions and revisions for M2

### PR DESCRIPTION
**Pull request by austin-hilberg**

### CODE REVIEW

File: docs-parts/admin/5-blob-config_lang3.rst [hunk 1]
```diff
@@ -1,3 +1 @@
-.. code-block:: python
-
-    >>> schema.external_table.clean_store('external-name')
+   This is done using the ``set_cache_folder`` method of the schema object.
```
Comment (dimitri-yatsenko) [inline docs-parts/admin/5-blob-config_lang3.rst]: What `set_cache_folder` method?

File: docs-parts/existing/1-Virtual-Modules_lang1.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+The function ``create_virtual_module`` of the ``dj.schema`` class provides access to virtual modules.
+It creates a python module with the given name from the name of a schema on the server, automatically adds classes to it corresponding to the tables in the schema.
+
+The function can take several parameters:
+
+  ``module_name``: displayed module name.
+
+  ``schema_name``: name of the database in MySQL.
+
+  ``create_schema``: if ``True``, create the schema on the database server.
```
Comment (dimitri-yatsenko) [inline docs-parts/existing/1-Virtual-Modules_lang1.rst]:
```suggestion
  ``create_schema``: if ``True``, create the schema on the database server if it does not already exist; if ``False`` (default), raise an error when the schema is not found.
```

File: docs-parts/existing/1-Virtual-Modules_lang1.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+  ``create_tables``: if ``True``, ``module.schema`` can be used as the decorator for declaring new classes.
```
Comment (dimitri-yatsenko) [inline docs-parts/existing/1-Virtual-Modules_lang1.rst]:
```suggestion
  ``create_tables``: if ``True``, ``module.schema`` can be used as the decorator for declaring new classes; if ``False``, such uses will raise an error -- the module is intend only to work with existing tables.
```

File: docs-parts/existing/1-Virtual-Modules_lang1.rst [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+The function returns the Python module containing classes from the schema object as well as the table classes.
```
Comment (dimitri-yatsenko) [inline docs-parts/existing/1-Virtual-Modules_lang1.rst]:
```suggestion
The function returns the Python module containing classes from the schema object with all the table classes already declared inside it.
```

File: docs-parts/queries/06-Restriction_lang4.rst [hunk 5]
```diff
@@ -1,11 +1,14 @@
...
+    # a list:
+    cond_list = ['first_name = "Aaron"', 'last_name = "Aaronson"']
 
-    Restriction by a collection, returning any entities matching any condition in the collection.
+    # a tuple:
+    cond_tuple = ('first_name = "Aaron"', 'last_name = "Aaronson"')
 
+    # a dataframe:
+    import pandas as pd
+    cond_frame = pd.DataFrame(tab.fetch())
```
Comment (dimitri-yatsenko) [inline docs-parts/queries/06-Restriction_lang4.rst]:
```suggestion
    cond_frame = pd.DataFrame(
                data={'first_name': ['Aaron'], 'last_name': ['Aarson']})
```

File: docs-parts/queries/06-Restriction_lang5.rst [hunk 6]
```diff
@@ -1,4 +1,11 @@
...
-``A & False`` and ``A - True`` are empty.
+.. code-block:: python
+
+    Student() & ['first_name = "Aaron"', 'last_name = "Aaronson"']
+
+.. figure:: ../_static/img/python_collection.png
+    :align: center
+    :alt: restriction by collection
+
+    Restriction by a collection, returning any entities matching any condition in the collection.
```
Comment (dimitri-yatsenko) [inline docs-parts/queries/06-Restriction_lang5.rst]:
```suggestion
    Restriction by a collection, returning all entities matching any condition in the collection.
```

File: docs-parts/admin/5-blob-config_lang3.rst (hunk 1 above)
Comment (austin-hilberg) [reply docs-parts/admin/5-blob-config_lang3.rst]: No idea. I only moved this line from the general content into a language-specific file.

Comment (austin-hilberg) [reply docs-parts/admin/5-blob-config_lang3.rst]: There does not seem to be such a method. Do you know what the method should be?

Comment (dimitri-yatsenko) [reply docs-parts/admin/5-blob-config_lang3.rst]:
```suggestion
This is done by saving the path in the ``"cache"`` key of the DataJoint configuration dictionary:

.. code-block:: python

   dj.config['cache'] = '/temp/dj-cache
```

**Pull request merged**